Assets/LeapMotion/Modules/InteractionEngine/Scripts/Editor/InteractionBehaviourEditor.cs

```diff
@@ -32,6 +35,105 @@ public class InteractionBehaviourEditor : CustomEditorBase<InteractionBehaviour>
                                 "_moveObjectWhenGrasped",
                                 "graspedMovementType",
                                 "graspHoldWarpingEnabled__curIgnored");
+
+      // Layer Overrides
+      specifyConditionalDrawing(() => target.overrideInteractionLayer,
+                                "_interactionLayer");
+      specifyConditionalDrawing(() => target.overrideNoContactLayer,
+                                "_noContactLayer");
+      specifyCustomDecorator("_noContactLayer", drawNoContactLayerDecorator);
+    }
+
+    private void drawInteractionManagerDecorator(SerializedProperty property) {
+      if (PrefabUtility.GetPrefabType(target) == PrefabType.Prefab) {
+        return;
+      }
+
+      bool shouldDrawInteractionManagerNotSetWarning = false;
+      foreach (var target in targets) {
```

Comment:
In general, if dealing with properties that are serialized, (like manager is I believe?) we should always favor using serialized properties over using the properties directly.  We handle multiple edit targets more easily, as well as better support for undo/edit detection/on validate ect...